pkg/system/system.go

```diff
@@ -47,6 +47,11 @@ type EnvGetter interface {
 	Env(key string) string
 }
 
+// LookupEnvGetter mocks out os.Getenv.
+type LookupEnvGetter interface {
```

Comment:
nit: `LookupEnvGetter` may sound like `LookupEnv` is a thing, and this interface can `Get` it, but it's not the case

```suggestion
type EnvLookuper interface {
```

Reply:
Given "Lookuper" isn't a word I chose a different name, but is updated now.